tests/helper/helper_generic.go

```diff
@@ -306,17 +311,55 @@ func CommonBeforeEach() CommonVar {
 	err := cfg.SetConfiguration(preference.ConsentTelemetrySetting, "false")
 	Expect(err).To(BeNil())
 	SetDefaultDevfileRegistryAsStaging()
+	// Ginkgo test related variables
+	commonVar.testFileName = strings.Replace(CurrentGinkgoTestDescription().FileName[strings.LastIndex(CurrentGinkgoTestDescription().FileName, "/")+1:strings.LastIndex(CurrentGinkgoTestDescription().FileName, ".")], "_", "-", -1) + ".go"
+	commonVar.testCase = CurrentGinkgoTestDescription().FullTestText
+	commonVar.testFailed = CurrentGinkgoTestDescription().Failed
+	commonVar.testDuration = CurrentGinkgoTestDescription().Duration.Seconds()
 	return commonVar
 }
 
 // CommonAfterEach is common function that cleans up after every test Spec (It)
 func CommonAfterEach(commonVar CommonVar) {
+	// Get details, including test result for each test spec and adds it to local testResults.txt file
+	var prNum string
+	var K8SorOcp string
+	var resultsRow string
+	prNum = os.Getenv("GIT_PR_NUMBER")
+	K8SorOcp = os.Getenv("KUBERNETES")
+	passedOrFailed := "PASSED"
+	if commonVar.testFailed {
+		passedOrFailed = "FAILED"
+	}
+	clusterType := "OCP"
+	if K8SorOcp == "KUBERNETES" {
```

Comment:
I think this should be `K8SorOcp == "true"`. Also, you can use the existing `IsKubernetesCluster()` helper function to decide on the cluster type.

Reply:
I am following your suggestion from https://github.com/redhat-developer/odo/pull/5449#discussion_r803475981